debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -170,7 +170,11 @@ protected void handleEvent(Event event) {
             } else {
                 rotateEventData = (RotateEventData) eventData;
             }
-            source.setBinlogStartPoint(rotateEventData.getBinlogFilename(), rotateEventData.getBinlogPosition());
+            String binlogFilename = rotateEventData.getBinlogFilename();
+            if (binlogFilename == null) {
+                logger.warn("Received 'null' binlog filename from rotate event: {}", event);                
```

Comment:
I'm not convinced that we should treat this as a warning, since IIUC a MySQL server operating normally can produce this in certain situations. Have you learned otherwise?